plugin/ipythonPlugins/src/dist/ipython/beaker_runtime.py

```diff
@@ -206,8 +208,9 @@ def transform(obj):
     return transformNaN(obj)
 
 def transformNR(obj):
-    if type(obj) == unicode:
-        return str(obj)
+    # this breaks unicode characters with non-ASCII symbols, default encoding is ASCII not utf8
```

Comment:
don't leave commented out code, just delete it.
put the explanation into the commit message.